Tests/TensorFlowTests/OperatorTests/MathTests.swift

```diff
@@ -58,6 +58,76 @@ final class MathOperatorTests: XCTestCase {
         testElementaryFunction(name: "root",
                                { x in root(x, 3) }, { x in Float.root(x, 3) })
     }
+    
+    func testAbs() {
+        let x = Tensor<Float>([-1.0])
+        let y = abs(x)
+        assertEqual(y, Tensor<Float>([1.0]))
+    }
+    
+    func testSquaredDifference() {
+        let x = Tensor<Float>([-5.8])
+        let y = Tensor<Float>([-5.7])
+        let z = squaredDifference(x, y)
+        let approxZ = 132.25
+        assertEqual(z, aproxZ, accuracy: 0.0001)
```

Comment:
@anigasan you forgot changing the variable name here.  Could you run `swift build` and `swift test` locally and tell if it passes or not?

Reply:
I'll check it out and see if it passes

Reply:
I made some small arithmetic mistakes which may have affected the build. I fixed them now